src/countBullsAndCows.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const { EXPECTED_NUMBER_LENGTH } = require('./constants');
+
+function countBullsAndCows(generatedNumber, userNumber) {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < EXPECTED_NUMBER_LENGTH; i++) {
+    if (generatedNumber.includes(userNumber[i])
```

Comment:
This check is repeated twice, you can create variable for `generatedNumber.includes(userNumber[i])`
The same with `generatedNumber.indexOf(userNumber[i]) === i`